string  word
float64 start_time
float64 end_time
